src/ImageRunModal.jsx

```diff
@@ -330,7 +330,7 @@ export class ImageRunModal extends React.Component {
             tempImage.isSystem = isSystem;
             tempImage.State = { Status: _("downloading") };
             tempImage.Created = new Date();
-            tempImage.Names = [tempImage.name];
+            tempImage.Name = tempImage.name;
```

Comment:
I'm not sure what this does. Changing property name/type can potentially break something somewhere, unless it's not used absolutely anywhere :-)

Reply:
That is expected, the property is Name in podman's API not Names.